enterprise/server/atime_updater/atime_updater_test.go

```diff
@@ -155,28 +155,25 @@ func waitExpectingUpdates(t *testing.T, expected []update, cas *fakeCAS) {
 	require.ElementsMatch(t, expected, cas.getUpdates())
 }
 
-func tickAlot(clock clockwork.FakeClock) {
+func tickAlot(clock clockwork.FakeClock, interval time.Duration) {
 	for i := 0; i < 25; i++ {
-		clock.Advance(time.Second)
+		clock.Advance(interval)
 		time.Sleep(5 * time.Millisecond)
 	}
```

Comment:
can't wait till we enable `testing/synctest` :P seems like it would be a perfect fit for this sort of thing